docs/logs: Update Logs Explorer Docs (#6406)
supabase/supabase: web/docs/guides/platform/logs.mdx

![SELECT query example](/img/guides/platform/logs/select-query.png)

> The log source, `edge_logs`, is required in order to formulate the SELECT query. Likewise, when querying the database logs, you would need to provide a table name reference as well. The list of supported product sources can be found under the **Sources** dropdown.

Reviewer: I'm not too sure what the these mean:

> The log source, `edge_logs`, is required in order to formulate the SELECT query. Likewise, when querying the database logs, you would need to provide a table name reference as well.

Perhaps just the last sentence is required

```suggestion
The list of supported product sources can be found under the **Sources** dropdown.
```

> Cross-source joining is currently supported.

Reviewer: seems like an unnecessary statement

Or, if you feel it is necessary: `Cross-source joining is supported.`. No need for `currently`

### Unnesting Arrays

> In the above example, in order to query across the metadata field, we need to unnest the field and add it as a join. When we inspect on an individual log by clicking on the log row, we can see that log metadata is stored as an array of objects. 

Reviewer: ```suggestion
To query the metadata in the above example, you can unnest the field and "join" the unnested data. Click on the log row and you can now see that log metadata is stored as an array of objects. 
```

#### Timestamp Behavior

> Each log timestamp is stored as a `TIMESTAMP` data type. In order to utilize the `timestamp` field in a query, you would need to use the appropriate [timestamp function](https://cloud.google.com/bigquery/docs/reference/standard-sql/timestamp_functions#timestamp) to query across the timestamp field.

Reviewer: ```suggestion
Each log entry is stored with a `timestamp`. In order to utilize the `timestamp` field in a query, you can use the appropriate [timestamp functions](https://cloud.google.com/bigquery/docs/reference/standard-sql/timestamp_functions#timestamp).
```

:::note

> Although timestamps are rendered as unix microsecond timestamps in the Log Explorer, SQL queries should use the `TIMESTAMP` data type only. If using a unix timestamp value in a query, cast the value to a `TIMESTAMP` data type.

Reviewer: ```suggestion
In the Log Explorer, timestamps are rendered as unix microsecond timestamps. SQL queries, however, should always use the `TIMESTAMP` data type. If you are using a unix timestamp value in a query, cast the value to a `TIMESTAMP` data type.
```

:::

## Templates

> Templates are available to help shorten the time needed to craft these queries. There are examples for each complexity level, which can serve as a foundation for your custom queries. Templates are available under the **Templates** tab, or under the **Templates** Dropdown in the **Query** tab.

Reviewer: ```suggestion
Templates are available to help craft you log queries. Templates are available under the **Templates** tab, or under the **Templates** Dropdown in the **Query** tab.
```
